.env_docker.example

```diff
@@ -26,5 +26,6 @@ SENDGRID_API_KEY=
 GOOGLE_MAPS_API_KEY=
 
 CIRCULAR_API_KEY=
+CIRCULAR_BASE_URL=
```

Comment:
is this new env var on docker compose file?

Reply:
yes